### PR TITLE
Update fungible_token_connector docs

### PR DESCRIPTION
**Pull request by fakela**

### CODE REVIEW

File: docs/bridge/connectors/fungible_token_connector.mdx [hunk 1]
```diff
@@ -3,69 +3,77 @@ title: Fungible Token Connector
...
+The fungible token connector consists of four contracts:
 
-The Fungible Token Connector consists of 4 contracts:
 - ft_connector
 - bridge_token_deployer
 - bridge_token
 - permissions_manager
 
-All of these contracts need to be installed on both chains.
+These contracts need to be installed on both the NEAR Testnet and Calimero shard.
```
Comment (ijerkovic) [inline docs/bridge/connectors/fungible_token_connector.mdx:13]:
```suggestion
These contracts need to be installed on both the NEAR Testnet and the Calimero private shard.
```

File: docs/bridge/connectors/fungible_token_connector.mdx [hunk 2]
```diff
@@ -3,69 +3,77 @@ title: Fungible Token Connector
...
+
+7. Prove the outcome, and if successful, the FTs are now unlocked on the source network.
+
+**Bridging between Calimero and NEAR when the original asset is on Calimero:**
+
+If the original asset exists only on Calimero and not on NEAR, bridging to NEAR will create a wrapped FT on NEAR. All steps from the previous sections apply, but in reverse.
+
+## Automatic bridging
+
+All of these steps are automated with the Calimero Bridge service. Users/DApps only need to make a `ft_transfer_call` to lock FTs on a chain, and the bridge service will handle the rest, automatically transferring the wrapped assets to the other chain. Similarly, when someone wants to unlock the funds, they simply need to make a `withdraw` call on the ft contract, and the bridge service will automatically update all the components. If the tokens were burnt on the starting chain, they will be unlocked on the other side.
```
Comment (ijerkovic) [inline docs/bridge/connectors/fungible_token_connector.mdx:79]:
```suggestion
**All of these steps are automated with the Calimero Bridge service**. The users of Calimero private shards do not need to make all the calls manually. Users/DApps only need to make a `ft_transfer_call` to lock FTs on a chain, and the bridge service will handle the rest, automatically transferring the wrapped assets to the other chain. Similarly, when someone wants to unlock the funds, they simply need to make a `withdraw` call on the ft contract, and the bridge service will automatically update all the components. If the tokens were burnt on the starting chain, they will be unlocked on the other side.
```

**Review by ijerkovic — APPROVED** [review]
2 comments

**Comment by fakela** [comment]
> 2 comments

fixed 👍

**Pull request merged**